Plutarch/Bool.hs

```diff
@@ -111,7 +111,7 @@ pif b case_true case_false = pmatch b $ \case
 
 -- | Boolean negation for 'PBool' terms.
 pnot :: Term s (PBool :--> PBool)
-pnot = phoistAcyclic $ plam $ \x -> pif x (pcon PFalse) $ pcon PTrue
+pnot = phoistAcyclic $ plam $ \x -> pif' # x # pconstant False # pconstant True
```

Comment:
Why use `pconstant` when it's unnecessarily longer? Just use `pcon PFalse` and such

Reply:
Habit, but yeah - changed.